src/detect-flowbits.c

```diff
@@ -400,16 +400,20 @@ static void DetectFlowbitsAnalyzeDump(const DetectEngineCtx *de_ctx,
         struct FBAnalyze *array, uint32_t elements);
 #endif
 
-void DetectFlowbitsAnalyze(DetectEngineCtx *de_ctx)
+int DetectFlowbitsAnalyze(DetectEngineCtx *de_ctx)
```

Comment:
Since only 2 values are ever returned, returning a `bool` would be more appropriate.

Reply:
I understand the point, however, when looking at other setup related code like DetectFlowbitSetup or other functions at detect-engine-build.c I felt it was more common to have a retval of 0 on success and -1 on failure (with existing code having 2 possible retvals too).

Reply:
It might be more common now but I feel it's more natural to use `bool` when the paradigm fits well.

I'm ok with it being as is but would prefer `bool`.